android/src/main/kotlin/com/bitmovin/player/flutter/FlutterPlayerView.kt

```diff
@@ -35,10 +37,33 @@ class FlutterPlayerView(
             messenger,
         )
 
+    private val playerView: PlayerView = PlayerView(context, player = null)
+
+    private val activity =
+        context.getActivity()
+            ?: error("Trying to create an instance of ${this::class.simpleName} while not attached to an Activity")
+
+    private val activityLifecycle =
+        (activity as? LifecycleOwner)
```

Comment:
The activity should always be null or a LifecycleOwner, right?
```suggestion
        (activity as LifecycleOwner?)
```

Reply:
The activity should never be null at this point, but it might not extend the `LifecycleOwner`.